python/tchannel/scheme.py

```diff
@@ -39,23 +39,25 @@ def serialize_body(obj)
 
         def deserialize_body(obj)
     """
-    def type(self):
-        raise NotImplementedError()
+    _type = "raw"
 
-    def serialize_header(self, obj):
-        raise NotImplementedError()
+    def type(self):
+        return self._type
 
     def deserialize_header(self, obj):
-        raise NotImplementedError()
+        return obj
 
-    def serialize_body(self, obj):
-        raise NotImplementedError()
+    def serialize_header(self, obj):
+        return obj
 
     def deserialize_body(self, obj):
-        raise NotImplementedError()
+        return obj
+
+    def serialize_body(self, obj):
+        return obj
 
 
-class JsonArgScheme(ArgScheme):
+class JsonArgScheme(RawArgScheme):
```

Comment:
Agreed. Either the interface is the same by accident, aka duck-typing, or it is the same on purpose. I prefer explicit, so I think you should use an `abc`. The classes that take an `ArgScheme` can enforce that the `abc` was implemented.